source/slang/hlsl.meta.slang

```diff
@@ -5042,8 +5283,22 @@ struct $(item.name)
         }
     }
 
+    /// Increment the object's hidden counter.
```

Comment:
Add that this function is currently not implemented for non HLSL targets.

Reply:
ack.